src/components/lib/VToast/VToast.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+import { useStore } from 'vuex'
+
+import CloseIcon from '@/components/icons/CloseIcon.vue'
+import VStack from '@/components/lib/VStack/VStack.vue'
+import type { ToastMessage } from '@/store/toast.store'
+
+const store = useStore()
```

Comment:
I think we should separate out store/business logic from the component because we may use this component on other projects

Reply:
Yeah that's the plan, right now using store was the easiest thing to implement in short time